nselib/httpspider.lua

```diff
@@ -633,7 +638,15 @@ Crawler = {
     o:loadLibraryArguments()
     o:loadDefaultArguments()
 
-    local response = http.get(o.host, o.port, '/', { timeout = o.options.timeout, redirect_ok = o.options.redirect_ok, no_cache = o.options.no_cache } )
+    --httpcookies library object
+    o.httpcookies = httpcookies.CookieJar:new()
+    local response
+
+    if o.options.docookies == true then
+      response = o.httpcookies:get(o.host, o.port, '/', { timeout = o.options.timeout, redirect_ok = o.options.redirect_ok, no_cache = o.options.no_cache } )
```

Comment:
Are you considering the case where a user defines the cookies to be used during crawling? Shouldn't be set here?